docs/README.md

```diff
@@ -23,35 +23,107 @@ WIP
 
 ```jsonc
 {
-  "modal-trigger#preview-shelf": {
+  "product-summary.shelf": {
     "children": [
-      "modal-layout#preview-shelf",
-      "product-summary-buy-button"
+      "stack-layout#prodsum",
+      "product-summary-name",
+      "product-rating-inline",
+      "product-summary-space",
+      "product-summary-price",
+      "add-to-cart-button"
     ]
   },
-  "rich-text#modal-title": {
+  "modal-trigger#quickview": {
+    "children": [
+      "modal-layout#quickview",
+      "rich-text#quickview"
+    ],
+    "props": {
+      "blockClass": "quickview"
+    }
+  },
+  "rich-text#quickview": {
     "props": {
-      "text": "## Preview modal title",
-      "blockClass": "myModalTitle"
+      "text": "Quickview",
+      "blockClass": "quickview"
     }
   },
-  "modal-header": {
+  "modal-layout#quickview": {
+    "props": {
+      "blockClass": "quickview",
+      "fullScreen": {
+        "phone": true,
+        "desktop": false,
+        "tablet": false
+      }
+    },
     "children": [
-      "rich-text#modal-title"
+      "flex-layout.row#quickview-main-row"
     ]
   },
-  "modal-content": {
+  "flex-layout.row#quickview-main-row": {
+    "children": [
+      "flex-layout.col#quickview-product-images",
+      "flex-layout.col#quickview-product-information"
+    ],
+    "props": {
+      "colGap": 4
+    }
+  },
+  "flex-layout.col#quickview-product-images": {
     "children": [
-      "all-my-children-elements-of-modal",
       "product-images"
-    ]
+    ],
+    "props": {
+      "rowGap": 0,
+      "width": "calc(50% - 0.375rem)"
+    }
   },
-  "modal-layout#preview-shelf": {
+  "flex-layout.col#quickview-product-information": {
     "children": [
-      "modal-header",
-      "modal-content"
-    ]
-  }
+      "vtex.store-components:product-name",
+      "product-rating-summary",
+      "product-price#quickview-price",
+      "product-separator",
+      "product-identifier.product",
+      "product-summary-sku-selector#quickview",
+      "link.product#quickview"
+    ],
+    "props": {
+      "preventVerticalStretch": true,
+      "rowGap": 0,
+      "width": "calc(50% - 0.375rem)"
+    }
+  },
+  "product-price#quickview-price": {
+    "props": {
+      "showInstallments": true,
+      "showSavings": true
+    }
+  },
+  "link.product#quickview": {
+    "props": {
+      "href": "/{slug}/p",
+      "label": "Mais detalhes >",
+      "blockClass": "quickview"
+    }
+  },
+  "product-summary-sku-selector#quickview": {
+    "props": {
+      "blockClass": "quickview"
+    }
+  },
+  "stack-layout#prodsum": {
+    "children": [
+      "product-summary-image",
+      "product-bookmark",
+      "product-summary-specification-badges",
+      "modal-trigger#quickview"
+    ],
+    "props": {
+      "blockClass": "shelf-summary"
+    }
+  },
 }
 ```
 
```

Comment:
This is for the section below:
Kind of a nitpick, but sort the CSS handles list.
If you don't have this VS Code extension yet, should make it pretty easy: `tyriar.sort-lines`